scripts/qa_crowbarsetup.sh

```diff
@@ -2723,6 +2704,9 @@ function custom_configuration
                 proposal_set_value database default "['attributes']['database']['ha']['storage']['drbd']['size']" "$drbd_database_size"
                 proposal_set_value database default "['deployment']['database']['elements']['database-server']" "['cluster:$clusternamedata']"
             fi
+            if iscloudver 8plus and [[ $want_database_sql_engine ]] ; then
```

Comment:
test failed with `Invalid database engine: .\n"` 
It should be && instead of `and`